Makefile

```diff
@@ -52,12 +52,22 @@ lint:
 deps:
 	make -C ./development deps
 
-reproducible-linux-create-image:
-	make -C ./reproducible/docker create-image
+reproducible-linux-create-image-vivid:
+	make -C ./reproducible/docker create-image-vivid
 
-reproducible-linux-build: reproducible-linux-create-image
+reproducible-linux-create-image-buster:
+	make -C ./reproducible/docker create-image-buster
+
+reproducible-linux-create-image: reproducible-linux-create-image-vivid
+
+reproducible-linux-build-vivid: reproducible-linux-create-image
 	make -C ./reproducible/docker build
 
+reproducible-linux-build-buster: reproducible-linux-create-image
+	make -C ./reproducible/docker build
+
+reproducible-linux-build: reproducible-linux-build-vivid
```

Comment:
My idea for the Makefile on the root directory is having only targets to perform tasks with default configurations. That's because the main Makefile was getting bigger and full of tasks not really related to building the executable.

`reproducible/Makefile` would have all the different flavors of reproducible and `Makefile` would only have a single `reproducible-linux-build` which can invoke `make -C ./reproducible/docker do-everything-for-default-reproducible`.

What do you think?

Actually the only reason I kept `reproducible-linux-build` and `reproducible-linux-create-image` there was for not surprising @olabini and because I was lazy to change the `REPRODUCIBILITY.md` to say './reproducible` directory is the place for all things reproducible.

Reply:
Makes sense! I'll update the Makefile.

Reply:
Lets's stick to the two make targets `reproducible-linux-create-image` and `reproducible-linux-build` as they were there before for now, ok ?

Reply:
Sure.